### ISSUE TITLE
The package page of a PIE-compatible extension should display the extension metadata

### ISSUE THREAD

**Issue by stof** [composer/packagist#1489]
Extensions (i.e. packages with type `php-ext` or `php-ext-zend`) have specific metadata in the `php-ext` field. It would be great to display this metadata (the extension name, whether they support ZTS or NTS builds of PHP, etc...) in the package display page.

**Comment by alcaeus** [comment]
Agree. At the moment, the package display page doesn't even show the extension name, so there's no way for people to connect an extension package to the old extension name people are used to.
